tests/network-setups/shared/corda/makefile

```diff
@@ -19,6 +19,6 @@ stop:
 	./scripts/stop-nodes.sh
 
 .PHONY: clean
-clean:	stop
+clean-local:	stop
```

Comment:
Clean for local build or artifactory(remote) build, is same. So I think `clean` would be better target name.

Reply:
@sanvenDev I reverted this to `clean` and changed in the `Getting Started` page from `clean-local` to `clean`. I wanted to ensure that it will be the same target name in both the `makefile` here and in the `Getting Started` page.